=== tests/cases/compiler/redeclareParameterInCatchBlock.ts ===
try {

} catch(e) {
>e : 任意
>类型别名 :=> 任意

    const e = null;
>e : 任意
>类型别名 :=> 任意
>null : 空值
>类型别名 :=> 空值
}

try {

} catch(e) {
>e : 任意
>类型别名 :=> 任意

    let e;
>e : 任意
>类型别名 :=> 任意
}

try {

} catch ([a, b]) {
>a : 任意
>类型别名 :=> 任意
>b : 任意
>类型别名 :=> 任意

    const [c, b] = [0, 1];
>c : 数字
>类型别名 :=> 数字
>b : 数字
>类型别名 :=> 数字
>[0, 1] : [数字, 数字]
>0 : 0
>1 : 1
}

try {

} catch ({ a: x, b: x }) {
>a : 任意
>类型别名 :=> 任意
>x : 任意
>类型别名 :=> 任意
>b : 任意
>类型别名 :=> 任意
>x : 任意
>类型别名 :=> 任意

}

try {

} catch(e) {
>e : 任意
>类型别名 :=> 任意

    function test() {
>test : () => 无值

        let e;
>e : 任意
>类型别名 :=> 任意
    }
}

